.kitchen.yml

```
@@ -66,6 +66,17 @@ platforms:
         - <%=ENV['ES_XPACK_LICENSE_FILE']%>:/tmp/license.json
         - /etc # This fixes certain java file actions that check the mount point. Without this adding users fails for some docker storage drivers
       run_command: "/sbin/init"
+  - name: ubuntu-20.04
+    driver_config:
+      image: ubuntu:20.04
+      privileged: true
+      provision_command:
+        - apt-get install -y -q ansible iproute2 python3-jmespath
```

```suggestion
        - apt-get install -y -q ansible gpg iproute2 python3-jmespath
```

Regarding https://devops-ci.elastic.co/view/Ansible/job/elastic+ansible-elasticsearch+pull-request/250/OS=ubuntu-2004,TEST_TYPE=custom-config,label=linux/console and https://docs.ansible.com/ansible/2.9/modules/apt_key_module.html, it seems that `apt_key` module requires `gpg` since 2.9 (ubuntu 18.04 install ansible 2.5.1 while ubuntu 20.04 install 2.9.6).